### ISSUE TITLE
Request for Multilingual Support in VOCR App

### ISSUE THREAD

**Issue by Balouisville** [chigkim/VOCR#25]
I hope this message finds you well. I am reaching out to express my appreciation for your work on the VOCR app, which has been a valuable resource for visually impaired individuals. However, I noticed that the app primarily supports English-speaking users, which limits its accessibility to non-English speaking communities.
I believe that extending the language support to include other languages could significantly enhance the app's usability and impact. It would provide non-English speaking visually impaired individuals with a tool to navigate their digital environments more effectively.
Here are a few suggestions on how this could be achieved:

1 .
Translation of the User Interface : Collaborate with volunteer translators or use translation platforms to translate the UI into various languages.
2 .
Multilingual Optical Character Recognition : Incorporate multilingual Optical Character Recognition systems to understand and interpret commands in different languages. Additionally, I suggest leveraging the built-in OCR engine of Mac to ensure support across both Intel and Silicon Mac systems.

I am eager to contribute to this initiative and can assist in coordinating translation efforts or providing feedback on multilingual support.
Thank you for considering my request. I am looking forward to the possibility of making VOCR a more inclusive and accessible tool for visually impaired individuals across the globe.
Warmest regards,

**Comment by chigkim** [comment]
MacOS VisionKit should detect language automatically and perform OCR based on the language it detected. I believe I enabled this feature.
It's not working?

**Comment by Balouisville** [comment]
No, it does not support multilingual optical character recognition.

**Comment by 645357696** [comment]
Hello, thank you very much for creating an OCR recognition program on the Mac system. I am Chinese and found that after using the hotkey scan, I can only see English and numbers, but not Chinese content. I hope to get your help. Thank you."
